.github/workflows/on-pull_request-opened-synchronize-reopened.yml

```diff
@@ -0,0 +1,22 @@
+name: pull_request-opened-synchronize-reopened-event
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  job:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out github-actions repo
+        uses: actions/checkout@v2
+        with:
+          repository: aerius/github-actions
+          path: aerius-github-actions
+          ref: v1.1
+
+      - uses: aerius/github-actions/events/pull_request-event-action@v1.1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ROOT_POM_DIRECTORY: repo
```

Comment:
```suggestion
```
This entry can go.. It's the default setting.. We only set it when the root pom isn't the root of the repository.